sklego/preprocessing.py

```diff
@@ -9,6 +9,94 @@
 from sklego.common import TrainOnlyTransformerMixin, as_list
 
 
+def _mk_average(xs, ys, intervals, span=1, method="average"):
+    """
+    asdafa
```

Comment:
This probably shouldn't be there

Reply:
strange ... the spellcheck should've picked it up.